src/ray/raylet/task_spec.cc

```diff
@@ -179,6 +179,21 @@ const ResourceSet TaskSpecification::GetRequiredResources() const {
   return ResourceSet(required_resources);
 }
 
+const ResourceSet TaskSpecification::GetRequiredPlacementResources() const {
+  auto message = flatbuffers::GetRoot<TaskInfo>(spec_.data());
+  auto required_resources = map_from_flatbuf(*message->required_resources());
+
+  auto it = required_resources.find(kCPU_ResourceLabel);
+  RAY_CHECK(it != required_resources.end());
+  // Actor creation tasks require one more CPU for placement purposes than they
+  // actually acquire during execution in order to account for the resource
+  // requirements of subsequent actor methods.
+  if (IsActorCreationTask()) {
```

Comment:
Is it always 1 more? What about the num_cpus=0 or num_cpus=N case?

Reply:
Oh good catch, in that case it should be 0. I'll fix this.

Reply:
That was a bug!